### PR TITLE
Add SF30 Pro 1.32 firmware

### PR DESCRIPTION
**Pull request by superm1**

### CODE REVIEW

File: releases/SF30_Pro-SN30_Pro/1.32/sf30sn30pro.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  <screenshots>
+    <screenshot type="default">
+      <image type="source">https://raw.githubusercontent.com/hughsie/8bitdo-firmware/master/screenshots/sn30-pro.svg</image>
+      <caption>Unplug the controller, hold down L1 + R1 + START until the red LED on top of the controller is flashing and then reconnect the controller.</caption>
+    </screenshot>
+  </screenshots>
+
+  <!-- only newer versions of fwupd know about the new controller -->
+  <requires>
+    <id compare="ge" version="1.0.3">org.freedesktop.fwupd</id>
```
Comment (hughsie) [inline releases/SF30_Pro-SN30_Pro/1.32/sf30sn30pro.metainfo.xml]: Is this true? Don't we need 1.3.3 for the wont-come-back stuff?

File: releases/SF30_Pro-SN30_Pro/1.32/sf30sn30pro.metainfo.xml [hunk 2]
```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2016 Richard Hughes <richard@hughsie.com> -->
+<!-- Copyright 2018 Mario Limonciello <mario.limonciello@dell.com> -->
```
Comment (hughsie) [inline releases/SF30_Pro-SN30_Pro/1.32/sf30sn30pro.metainfo.xml]: 2019?

File: releases/SF30_Pro-SN30_Pro/1.32/sf30sn30pro.metainfo.xml (hunk 1 above)
Comment (hughsie) [reply releases/SF30_Pro-SN30_Pro/1.32/sf30sn30pro.metainfo.xml]: True.

**Comment by superm1** [comment]
Can you get this onto LVFS too?  Not sure how it usually flows for 8bitdo stuff.

**Comment by hughsie** [comment]
It should appear in `testing` in a few hours.

**Pull request merged**